rightscale/rsc/client.go

```diff
@@ -655,7 +662,13 @@ func processOutputs(res interface{}) map[string]interface{} {
 	outputs := make(map[string]interface{}, len(outs))
 	for _, out := range outs {
 		om := out.(map[string]interface{})
-		outputs[om["name"].(string)] = om["value"].(map[string]interface{})["value"]
+		value := om["value"].(map[string]interface{})["value"]
+		stringValue, ok := value.(string)
```

Comment:
Go pro-tip: do not use mix case local variable names. Local variable names should be "short" (ideally one or two characters).